apps/examples/src/examples/remove-tools-actions/RemoveToolsActionsExample.tsx

```diff
@@ -0,0 +1,32 @@
+import { TLUiOverrides, Tldraw } from 'tldraw'
+
+const overrides: TLUiOverrides = {
+	actions(editor, actions) {
+		// remote actions - see the action bar
+		delete actions.delete
+		delete actions.duplicate
+		delete actions.undo
+		delete actions.redo
+
+		return actions
+	},
+	tools(editor, tools) {
+		// remove tools - see the toolbar
+		delete tools.draw
```

Comment:
I hesitate that we should make this an example. Here's why:
I wouldn't want to encourage this pattern b/c it might lead the developer down a whack-a-mole path. Later when we add, say, a sticker tool, they'll have to keep coming back and pro-actively deleting new things as our surface area changes. 

Basically, I think we should encourage an example that shows the whitelist path (i.e. I want these 5 [hand, select, draw] tools).